portfolio/src/main/webapp/leave-comment/script.js

```diff
@@ -0,0 +1,37 @@
+function getComments() {
+  console.log("Downloading comments.");
+  const responsePromise = fetch("/get-comments");
+  responsePromise.then(handleResponse);
+}
+
+function handleResponse(response) {
+  console.log('Handling the response.');
+
+  // response.text() returns a Promise, because the response is a stream of
+  // content and not a simple variable.
+  const textPromise = response.text();
+  const JSONPromise = textPromise.then(JSON.parse);
+  // When the response is converted to text, pass the result into the
+  // addCommentsToDom() function.
+  JSONPromise.then(addCommentsToDom);
+}
+
+function addCommentToHTML(commentJSON) {
+  $('#comment-list')
```

Comment:
is jquery recommended in the tutorial? most tasks can be done by modern JS in the meantime.